app/controllers/auth/registrations_controller.rb

```diff
@@ -10,4 +12,10 @@ def new
     @have_users = User.any?
     super
   end
+
+  def restrict_admin
```

Comment:
Could you add documentation? You don't have to be too verbose, just an explanation of what it does.